packages/components/react/src/index.ts

```diff
@@ -1,10 +1,8 @@
 export * from './BootScreen.js';
-export * from './CodeMirrorEditor/index.js';
-export * from './FileTree.js';
```

Comment:
Removed those exports because they're not used at the moment and the intent is to have them imported via the `/core` module.